bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -0,0 +1,37 @@
+{% extends "bare.html" %}
+{% stylesheet 'firefox_new_firstrun' %}
+{% block site_js %}
+  {% javascript 'site' %}
```

Comment:
This `site` bundle is currently being loaded twice, as it's already being included in the `base-resp.html` template which `bare.html` extends. You can remove this line.